cmd/tink-cli/cmd/get/get.go

```diff
@@ -0,0 +1,100 @@
+package get
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/jedib0t/go-pretty/table"
+	"github.com/spf13/cobra"
+)
+
+type CmdOpt struct {
+	// Headers is the list of headers you want to print as part of the list
+	Headers []string
+	// RetrieveData reaches out to Tinkerbell and it gets the required data
+	RetrieveData func(context.Context) ([]interface{}, error)
+	// PopulateTable populates a table with the data retrieved with the RetrieveData function.
+	PopulateTable func([]interface{}, table.Writer) error
+
+	// Output specifies the output you want the list of resources printed
+	// out. By default it is table but it can be JSON ar CSV.
+	Output string
+	// NoHeaders does not print the header line
+	NoHeaders bool
+}
+
+const shortDescr = `
+Display one or many resources`
+
+const longDescr = `
+Prints a table contains the most important information about a specific
+resource. You can specify the kind of output you want to receive. It can be
+table or csv or json.
+`
+
+const exampleDescr = `
+# List all hardware in table output format.
+tink hardware get
+
+# List all workflow in csv output format.
+tink template get -o csv
```

Comment:
`-f` is file in create or whatever I don't remember. I used `-o` because kubectl uses it. Docker uses `--format`, I am happy to use it but without the short version.